functions/java-jre.bash

```diff
@@ -120,131 +157,113 @@ java_zulu_stable(){
 
   if [ -n "$INTERACTIVE" ]; then
     if ! (whiptail --defaultno --title "Untested Version of Java" --no-button "Back" --yes-button "Continue" --yesno "$updateText" 11 80); then echo "CANCELED"; return 0; fi
-    cond_redirect java_zulu_fetch "$1"
-    cond_redirect java_zulu_install
   else
-    echo "$(timestamp) $consoleText"
-    cond_redirect java_zulu_fetch "$1"
-    cond_redirect java_zulu_install
+    echo "$(timestamp) [openHABian] $consoleText"
   fi
+  cond_redirect java_zulu_fetch "$1"
+  cond_redirect java_zulu_install "$1"
 }
 
 ## Install Java Zulu directly from fetched files
 ## Valid arguments: "Zulu8-32", "Zulu8-64", "Zulu11-32", or "Zulu11-64"
 ##
 ##    java_zulu_install(String arch)
 ##
-java_zulu_install(){
+java_zulu_install() {
+  local jdkArch
   local jdkBin
   local jdkLib
-  local jdkArch
 
-  cond_redirect systemctl stop openhab2.service
+  if [ -z "$UNATTENDED" ] && [ -z "$BATS_TEST_NAME" ]; then
```

Comment:
don't like to build in different behavior under test, but in this case it seems necessary...

Reply:
Yeah, it was the best way to avoid errors. In this circumstance it appears to be the best option.